src/SDKs/Search/DataPlane/Search.Tests/Tests/SkillsetsTests.cs

```diff
@@ -0,0 +1,610 @@
+                Assert.Equal(HttpStatusCode.NoContent, deleteResponse.Response.StatusCode);
+
+                deleteResponse = searchClient.Skillsets.DeleteWithHttpMessagesAsync(skillset.Name).Result;
+                Assert.Equal(HttpStatusCode.NotFound, deleteResponse.Response.StatusCode);
+            });
+        }
+
+        private void CreateAndGetSkillset(SearchServiceClient searchClient, Skillset expectedSkillset)
+        {
+            try
+            {
+                searchClient.Skillsets.Create(expectedSkillset);
+                Skillset actualSkillset = searchClient.Skillsets.Get(expectedSkillset.Name);
+                AssertSkillsetEqual(expectedSkillset, actualSkillset);
+            }
+            finally
+            {
+                searchClient.Skillsets.Delete(expectedSkillset.Name);
+            }
+        }
+
+        private static Skillset CreateTestOcrSkillset(int repeat, string name = null, bool shouldDetectOrientation = false)
+        {
+            List<Skill> skills = new List<Skill>();
+
+            List<InputFieldMappingEntry> _inputs = new List<InputFieldMappingEntry>()
+            {
+                new InputFieldMappingEntry { Name = "url", Source = "/document/url" },
+                new InputFieldMappingEntry { Name = "queryString", Source = "/document/queryString" }
+            };
+
+            for (int i = 0; i < repeat; i++)
+            {
+                List<OutputFieldMappingEntry> _outputs = new List<OutputFieldMappingEntry>()
+                {
+                    new OutputFieldMappingEntry { Name = "text", TargetName = "mytext" + i }
+                };
+
+
+                skills.Add(new OcrSkill("Tested OCR skill", PathSeparator + RootName, _inputs, _outputs)
+                {
+                    TextExtractionAlgorithm = TextExtractionAlgorithm.Printed,
+                    ShouldDetectOrientation = shouldDetectOrientation,
+                    DefaultLanguageCode = "en"
+                });
+            }
+
+            return new Skillset(name: name ?? SearchTestUtilities.GenerateName(), description: "Skillset for testing OCR", skills: skills);
+        }
+
+        private void CreateAndValidateSkillset(SearchServiceClient searchClient, Skillset expectedSkillset)
+        {
+            try
+            {
+                Skillset actualSkillset = searchClient.Skillsets.Create(expectedSkillset);
+                AssertSkillsetEqual(expectedSkillset, actualSkillset);
+            }
+            finally
+            {
+                searchClient.Skillsets.Delete(expectedSkillset.Name);
+            }
+        }
+
+        private static void AssertSkillsetEqual(Skillset expected, Skillset actual)
+        {
+            Assert.Equal(expected, actual, new ModelComparer<Skillset>());
+        }
+
+        private static Skillset CreateTestSkillsetOcrKeyPhrase(string ocrLanguageCode = "en", string keyPhraseLanguageCode = "en")
```

Comment:
Why use string parameters when you could use the custom types you created? That way you don't have to call `.ToString()` in your test cases.

Reply:
Fixed. Thanks.